packages/core/core-flows/src/cart/steps/compensate-payment-if-needed.ts

```diff
@@ -51,20 +48,7 @@ export const compensatePaymentIfNeededStep = createStep(
       }
     )
 
-    if (paymentSession.status === PaymentSessionStatus.AUTHORIZED) {
-      try {
-        await paymentModule.cancelPayment(paymentSession.id)
-      } catch (e) {
-        logger.error(
-          `Error was thrown trying to cancel payment session - ${paymentSession.id} - ${e}`
-        )
-      }
-    }
-
-    if (
-      paymentSession.status === PaymentSessionStatus.CAPTURED &&
-      paymentSession.payment?.id
-    ) {
+    if (paymentSession.payment?.captured_at) {
```

Comment:
comment: should we add to docs that once payment is refunded in the autocapture flow if cart complete fails, a new session needs to be initialized